docs/content/configs/recommended-custom.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: recommended-custom
```

Comment:
And also add config to `docs/utils/pages.ts`?

Reply:
@azat-io This was added in f40baa2e3659992b04010b91ad7094765f170227, unless I missed something 🙂